### DISCUSSION TITLE
Search in multireddit

### DISCUSSION THREAD

**Discussion by Yaurok** [Docile-Alligator/Infinity-For-Reddit#899] [Ideas]
I love everything about this app and I'd like to see implemented this feature which I usually use a lot on my time at reddit: let the user search something in a multireddit of their choice, so that what they look for get searched in all the subreddits inside that specific multireddit

Useful when you need to find something and you made a multireddit of similar subjects; you can increase the chance of getting the post you need (and saving time doing so) among all those instead of only one subreddit at a time

If you can get this done I'd really appreciate! Thanks for the hard work on this app!

**Comment by Docile-Alligator** [comment]
Good suggestion! This is a little bit hard to implement but I will try.

**Comment by karan** [comment]
Would appreciate this

**Comment by VariabileAleatoria** [comment]
any news?

**Comment by peterorparker** [comment]
any possibility this might get implemented?

**Comment by Xyde** [comment]
With NSFW content only available through Multireddit, me thinks this should be higher priority...
